Fix wrong command in Tm1637.DisplayRaw(byte, byte)

smdn opened the pull request:
`Tm1637.DisplayRaw(byte segmentAddress, byte rawData)` is sending incorrect command.
With current implementation, it always displays `rawData` to the first segment, regardless of the parameter `segmentAddress`. (See repro code and image below)

`DisplayRaw` must send `DataCommand.AddressCommandSetting + segmentAddress` to set the display address, not `DataCommand.FixAddress + segmentAddress`.

Repro code:
```cs
using System;
using System.Device.Gpio;
using Iot.Device.Tm1637;

class Program
{
  static void Main(string[] args)
  {
    var display = new Tm1637(
      pinDio: 16,
      pinClk: 18,
      pinNumberingScheme: PinNumberingScheme.Board
    );

    display.Brightness = 7;
    display.ScreenOn = true;
    display.ClearDisplay();

    display.Display(0, Character.Digit0);
    display.Display(1, Character.Digit1);
    display.Display(2, Character.Digit2);
    display.Display(3, Character.Digit3);
    display.Display(4, Character.Digit4);
    display.Display(5, Character.Digit5);
  }
}
```

This code displays characters like below.
![TM1637-broken](https://user-images.githubusercontent.com/553926/71270996-451fcd00-2396-11ea-99b9-3afcaf805ffe.jpg)

Fixed implementation displays characters like below as expected.
![TM1637-fixed](https://user-images.githubusercontent.com/553926/71271005-4bae4480-2396-11ea-8bbb-48a9be7af132.jpg)

krwq commented:
cc: @Ellerbach

krwq commented:
@smdn sanity check, have you verified the original sample also works correctly after this change?

smdn commented:
The original sample does not call `Display(byte, Character)`.
It calls only `Display(ReadOnlySpan<Character>)`.
This change does not affect to the sample.

And just to be sure, I attempted to run the sample for verification, but it throws an exception😕
The sample has another probrem. I will open another pull request later.

smdn commented:
Please see  #913

krwq commented:
@smdn I'm testing this on 4-segment the sample shows following for me:
- (as expected) 4238
- (as expected) some random segments
- (as expected) 2105
- 3 digits of the time - I think this is a sample issue (it should not use `5` in the first 4 elements of SegmentOrder since in my case I don't have a 5th character and it displays as blank)

also sample should be updated so that it also covers the API being tested here (i.e. before testing segment order add something to cover the API you're fixing here)

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/iot?pullRequest=910) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/iot?pullRequest=910) before we can accept your contribution.<br/><br/>:x: smdn [sign now](https://cla.dotnetfoundation.org/dotnet/iot?pullRequest=910)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/iot?pullRequest=910) it.</sub>

smdn commented:
@krwq I updated sample project and tested with 6-segment display.
Added sample works fine for me.
But sample of `Display(ReadOlnySpan<Character>)` displays character at unexpected position as I commented at #913.

-`Display(ReadOnlySpan<Character>)` [`42.38  `] : ✅ as expected
-`Display(byte, Character)` [`012345`] (added sample) : ✅ as expected
-`Display(ReadOnlySpan<Character>)` some random segments : ❌ characters are as expected, but position is wrong
-`Display(ReadOnlySpan<Character>)` [`210543`] : ❌ displays [`105432`]
-`Display(byte, Character)` [`654987`] (added sample) : ✅ as expected
-`Display(ReadOnlySpan<Character>)` [`012345`] with blinking (updated sample) : ✅ as expected
-`Display(ReadOnlySpan<Character>)` [`mm.ss  `] (updated sample) :  ✅ as expected

krwq commented:
@smdn there seem to be a lot of irrelevant changes in this PR, can you clean this up please?

smdn commented:
Cleaned up. Sorry for bothering you.
This sample will throw exception because changes of #913 is not merged yet. Is this OK?

krwq commented:
@smdn not sure I understand the question... #913 seem to be merged a while ago...

smdn commented:
@krwq If there is no probrems for you, forget about it.
I just concerned that the sample cannot be run on this branch.

krwq commented:
@smdn can you clarify, is there any work left for the sample to be run correctly? Last time I checked - before this PR the sample worked fine on 4 digit segment

smdn commented:
@krwq

1. To run the sample, we need to fix some code.

I've reported and fixed this at #913. It is already merged to `master` and closed.
But on this branch it have not be merged. The sample still throws exception.


2. To display characters correctly, it might be needed to fix `Display(ReadOnlySpan<Character>)`.

With my 6 digit segment, `Display(ReadOnlySpan<Character>)` displays character on unexpected position.
I've reported this problem at comment below.

- https://github.com/dotnet/iot/pull/910#issuecomment-568467584
- https://github.com/dotnet/iot/pull/913#discussion_r360694428

I have not found out the cause of this problem yet.


Should I open new issue about problem of `Display(ReadOnlySpan<Character>)`?
Or should I fix it on this PR?

krwq commented:
@smdn, last time I was checking your PRs I manually merged both fixes... Also you can rebase this branch to get latest changes

smdn commented:
Rebase done.
Also confirmed the sample runs without throwing exception.

joperezr commented:
Thanks for this @smdn

smdn commented:
Thank you for merging!
I am glad to contribute to .NET world😊

joperezr commented:
😄  Let us know if you are interested in fixing more issues and we'll gladly look for great issues you can contribute with.
